hknweb/tutoring/models.py

```diff
@@ -90,37 +90,38 @@ def end_time(self):
         return self.time(self.hour + 1)
 
     def __repr__(self):
-        return "TimeSlot(day={}, start={}, end={})".format(self.get_day(), self.start_time(), self.end_time())
+        return "TimeSlot(day={}, start={}, end={})".format(
+            self.get_day(), self.start_time(), self.end_time()
+        )
+
     def __str__(self):
-        return str(self.get_day()) + ' ' + str(self.start_time()) + ' to ' + str(self.end_time())
+        return (
+            str(self.get_day())
+            + " "
+            + str(self.start_time())
+            + " to "
+            + str(self.end_time())
+        )
+
 
 class Room(models.Model):
     DEFAULT_ROOM_CHOICES = [
-        # (0, 'Cory', '290'),
-        # (1, 'Soda', '345'),
-        (0, 'Online', '')
+        (0, "Hybrid/Cory", "290"),
+        (1, "Hybrid/Soda", "345"),
+        (2, "Online", ""),
     ]
     building = models.CharField(max_length=255)
     room_num = models.CharField(max_length=255, blank=True)
 
     def get_num_building(self):
-        room_num_str = ""
-        if self.room_num:
-            room_num_str = self.room_num + " "
-        return room_num_str + self.building
+        return "%s %s" % (self.building, self.room_num)
```

Comment:
If there was no room number, the extra space should someone find it just looks weird